### DISCUSSION TITLE
"10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog" couldn't be opened" when attempting to install macOS 13.5.2 (v.32)

### DISCUSSION THREAD

**Discussion by macinthass** [grahampugh/erase-install#435] [General]
We are using the erase-install script to install macOS system updates with a single click via Nudge (JAMF Pro). I currently see the script fail on a number of Macs with the following error:

_The file "index-13-12-10.16-10.15-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog"couldn't be opened._

Is that even the proper file? I am concerned because it mentions a bunch of old macOS versions but perhaps that is all good? Any ideas why it fails? I will include the entire log data below. Thanks for your help!

[STEP 1 of 5]
--
Executing Policy Software Update (EraseInstall)
[STEP 2 of 5]
Running script Software-Update (Self-Service) - EraseInstall...
Script exit code: 0
Script result: User yuetianl has now admin rights
[STEP 3 of 5]
Running script Erase Install (version 30.2)...
Script exit code: 1
Script result: 2023-09-11 20:08:58 \| v30.2 \| [erase-install] v30.2 script execution started: Mon Sep 11 20:08:58 PDT 2023  2023-09-11 20:08:58 \| v30.2 \| [erase-install] Arguments provided: / Q-C02C76U0MD6R-yuetianl yuetianl --reinstall --no-fs       2023-09-11 20:08:58 \| v30.2 \| [erase-install] System version: 13.5.1 (Build: 22G90) 2023-09-11 20:08:59 \| v30.2 \| [erase-install] Making working directory at /Library/Management/erase-install 2023-09-11 20:08:59 \| v30.2 \| [erase-install] Making log directory at /Library/Management/erase-install/log 2023-09-11 20:08:59 \| v30.2 \| [check_for_swiftdialog_app] Downloading swiftDialog...   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current                                  Dload  Upload   Total   Spent    Left  Speed    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0   51 6100k   51 3160k    0     0  3652k      0  0:00:01 --:--:--  0:00:01 3652k 100 6100k  100 6100k    0     0  5974k      0  0:00:01  0:00:01 --:--:-- 18.5M installer: Package name is  installer: Installing at base path / installer: The install was successful. 2023-09-11 20:09:03 \| v30.2 \| [check_for_swiftdialog_app] swiftDialog is installed 2023-09-11 20:09:03 \| v30.2 \| [check_for_swiftdialog_app] Creating dialog log (/var/tmp/dialog.Piy)... 2023-09-11 20:09:03 \| v30.2 \| [log_rotate] Start rotating logs in /Library/Management/erase-install/log 2023-09-11 20:09:03 \| v30.2 \| [log_rotate] Finished rotating logs in /Library/Management/erase-install/log 2023-09-11 20:09:03 \| v30.2 \| [erase-install] Caffeinating this script (pid=29469) 2023-09-11 20:09:03 \| v30.2 \| [check_free_space] OK - 836 GB free/purgeable disk space detected 2023-09-11 20:09:03 \| v30.2 \| [erase-install] Looking for existing installer app or pkg 2023-09-11 20:09:03 \| v30.2 \| [find_existing_installer] No valid installer found. 2023-09-11 20:09:03 \| v30.2 \| [erase-install] Running on architecture i386 2023-09-11 20:09:03 \| v30.2 \| [get_default_dialog_args] Invoking utility dialog 2023-09-11 20:09:03 \| v30.2 \| [check_for_mist] Downloading mist-cli...   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current                                  Dload  Upload   Total   Spent    Left  Speed    0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0   25 2190k   25  558k    0     0  1158k      0  0:00:01 --:--:--  0:00:01 1158k 100 2190k  100 2190k    0     0  4003k      0 --:--:-- --:--:-- --:--:-- 24.8M installer: Package name is mist-cli installer: Installing at base path / installer: The install was successful. 2023-09-11 20:09:06 \| v30.2 \| [check_for_mist] mist is installed 2023-09-11 20:09:06 \| v30.2 \| [get_mist_list] Non-default catalog selected (darwin version 22) ┌──────────────────┐ │ INPUT VALIDATION │ └──────────────────┘   ├─ Search only for latest (first) result will be 'false'...   ├─ Include betas in search results will be 'false'...   ├─ Only include compatible installers will be 'true'...   ├─ Export path will be '/Library/Management/erase-install/mist-list.json'...   ├─ Export path file extension is valid...   ├─ Output type will be 'ascii'... ┌────────┐ │ SEARCH │ └────────┘   ├─ Searching for macOS Installer versions...   ├─ The file “index-13-12-10.16-10.15-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog” couldn’t be opened.   ├─ Exported list as JSON: '/Library/Management/erase-install/mist-list.json'   └─ Found 0 macOS Installer(s) available for download  Path not found. 2023-09-11 20:09:07 \| v30.2 \| [run_mist] ERROR: mist was unable to locate any installers (probably no internet connection)  2023-09-11 20:09:07 \| v30.2 \| [erase-install] attempting to terminate the 'caffeinate' process - Termination message indicates success  2023-09-11 20:09:07 \| v30.2 \| [finish] sending quit message to dialog (/var/tmp/dialog.Piy) 2023-09-11 20:09:08 \| v30.2 \| [finish] Script exit code: 1
Error running script: return code was 1.
[STEP 4 of 5]
[STEP 5 of 5]

**Comment by grahampugh** [comment]
Hi there, yes, that is the legitimate URL of the catalog, and it's working fine for me. You may want to try again and/or check if you have some network software preventing access to https://swscan.apple.com
